build_n_deploy/naiserator/naiserator_prod_gcp.yaml

```diff
@@ -67,7 +67,7 @@ spec:
       memory: 512Mi
       cpu: 200m
 
-  redis:
+  valkey:
```

Comment:
Må oppdatere config også:
`... .env.REDIS_URI_SESSIONS` -> `... .env.VALKEY_URI_SESSIONS`

Samme for `REDIS_USERNAME_SESSIONS` og `REDIS_PASSWORD_SESSIONS`